extensions/amp-form/0.1/amp-form.js

```diff
@@ -109,6 +116,39 @@ export class AmpForm {
     }
     this.form_.classList.add(`amp-form-${state}`);
   }
+
+  /**
+   * @param {string} state
+   * @param {!Object} data
+   * @private
+   */
+  renderTemplate_(state, data) {
+    const container = this.form_.querySelector(`[${state}]`);
+    if (container) {
+      // TODO(#3587): Move the cleanup to do during submission.
+      this.cleanupRenderedTemplate_(state);
+      return this.templates_.findAndRenderTemplate(container, data)
+          .then(rendered => {
+            rendered.classList.add(`${state}-message`);
```

Comment:
Do we have to have this class? Is there any way to do it some other way? E.g. we can use `i-amp-rendered` attribute or such? This way we won't have any ambiguity of conflict against the class name?